tests/test_shell.py

```diff
@@ -64,8 +65,24 @@ def call(self, command, input=None, shell=False):
         out, err = p.communicate()
         if p.returncode != 0:
             raise ExitCodeError("STDOUT='{}' STDERR='{}'".format(out, err))
+        if error:
+            return err.decode()
         return out.decode()
 
+    def test_print_usage(self):
+        with pytest.raises(ExitCodeError):
+            out = self.call('python -m signac'.split())
+            assert 'usage:' in out
```

Comment:
Considering that `pytest.raises(..)` is implying that we assume that the `call()` is raising an `ExitCodeError`, would this assert never be tested?

Reply:
Sorry, I didn't realize that earlier. will change.